src/test/resources/PotoracMihai/FirstUITest.feature

```diff
@@ -0,0 +1,14 @@
+Feature: First UI Test
+
+  #Verify page title is “Ecommerce Website Template”
+
+Background: Open Page
+
+
+Given Shopify Page is Open PM
```

Comment:
Only hopify Page is Open PM should be under "Background: Open Page"

two other steps should be under Scenario